Easydict/App/AppDelegate.h

```diff
@@ -7,10 +7,7 @@
 //
 
 #import <Cocoa/Cocoa.h>
-#import <Sparkle/SPUStandardUpdaterController.h>
 
 @interface AppDelegate : NSObject <NSApplicationDelegate>
 
-@property (weak) IBOutlet SPUStandardUpdaterController *updaterController;
```

Comment:
Since you have removed this, some setup code in `AppDelegate.m` for `SPUStandardUpdaterController `  could be removed too.